addons/mail/static/src/core/messaging_service.js

```diff
@@ -235,7 +235,7 @@ export class Messaging {
                     const command = rtcSessions[0][0];
                     this._updateRtcSessions(thread.id, sessionsData, command);
 
-                    if (invitedByUserId !== this.store.user?.user.id) {
+                    if (invitedByUserId !== this.store.user?.id) {
```

Comment:
id compare on persona should always include compare on type too?

Reply:
Looks correct to check `store.user?.id === invitedByUserId`: should always show "You have been invited" when being invited by internal user. `store.user.id` makes only sense when self is user.

Only thing missing here is `invitedByUserId` being set. I'm adding it, otherwise guests receiving this notification without being invited by a user can wrongly be considered an invitation by a user.